CP04OSSM/CP04OSSM_R00013_ingest.csv

```diff
@@ -0,0 +1,46 @@
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl27/ctdbp1/*.ctdbp1.log,CP04OSSM-RID27-03-CTDBPC000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/instruments/dcl27/CTDBP_sn*/SBE*.hex,CP04OSSM-RID27-03-CTDBPC000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl27/dosta1/*.dosta1.log,CP04OSSM-RID27-04-DOSTAD000,recovered_host,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/cpm1/irid2shore/cpm_status*.txt,CP04OSSM-SBC11-00-CPMENG000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/syslog/*.syslog.log,CP04OSSM-SBD11-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/mopak/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/hyd1/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/metbk/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl11/fdchp2/*fdchp2.log,CP04OSSM-SBD11-08-FDCHPB000,recovered_host,Available,test deployment; R3-50 anemometer
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/instruments/dcl11/FDCHP_sn*/D20*/fdchp*.dat,CP04OSSM-SBD11-08-FDCHPB000,recovered_inst,Available,test deployment; R3-50 anemometer
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/hyd2/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/pco2a/*pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/cg_data/dcl12/fdchp1/*fdchp1.log,CP04OSSM-SBD12-08-FDCHPA000,recovered_host,Available,test deployment
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00013/instruments/dcl12/FDCHP_sn*/D20*/fdchp*.dat,CP04OSSM-SBD12-08-FDCHPA000,recovered_inst,Available,test deployment
```

Comment:
looks good - data available & aligns w/ data recovery checklist; no pco2w instrument data as noted on data recovery checklist & line commented out.

Reply:
Looks good, except that FDCHPs should be commented out (test deployment)